docs/core/integration-quality-scale/rules/iqs014.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "IQS014 - Entities use device classes where possible"
+---
+
+IQS014 is as follows:
+
+Entities use device classes where possible
+
+## Reasoning
+
+Device classes are a way to give context to an entity.
+These are used by Home Assistant for various purposes like:
```

Comment:
They are also used to adjust the representation of the entity in the Home Assistant UI.